charts/meep-wais/values-template.yaml

```diff
@@ -55,6 +55,8 @@ ingress:
         {{- end }}
   annotations:
     kubernetes.io/ingress.class: nginx
+    nginx.ingress.kubernetes.io/proxy-read-timeout: "3600"
```

Comment:
This is required to prevent websocket connection timeouts when there are no notifications being sent. This value must be greater than the websocket keepalive time that is currently set to 10 min.